lib/src/validator/language_version.dart

```diff
@@ -42,6 +42,7 @@ class LanguageVersionValidator extends Validator {
     for (final path in ['lib', 'bin']
         .map((path) => entrypoint.root.listFiles(beneath: path))
         .expand((files) => files)
+        .map(p.normalize)
```

Comment:
@sigurdm, do we have this issue elsewhere? Maybe in `pana`?

Reply:
Good question
We don't normalize the path here: https://github.com/dart-lang/pana/blob/master/lib/src/tag_detection.dart#L164

On the other hand the paths there are constructed differently. Not sure it's an issue

Reply:
I created https://github.com/dart-lang/pub/pull/2409 to make sure we don't have the problem in pub anymore